webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -2331,11 +2338,17 @@ object RudderConfig extends Loggable {
   ////////////////////// Snippet plugins & extension register //////////////////////
   lazy val snippetExtensionRegister: SnippetExtensionRegister = new SnippetExtensionRegisterImpl()
 
+  private[this] lazy val cachedNodeConfigurationService = {
+    val cached = new CachedNodeConfigurationService(findExpectedRepo, nodeInfoServiceImpl)
+    cached.init().runNow
+    cached
+  }
+
   /*
```

Comment:
is it possible to completly hide findExpectedRepo from public use with a facade that delegate to either it or cache ? We would be sure to never call the bad service

Reply:
it is a private[this] - i'm unsure how to do better than that (unless putting a big comment around it to not use it)

Reply:
or it is used twice, once per cache, so I could instanciate one per cache, overhead should be minimal